### ISSUE TITLE
Multi-account Container Colors

### ISSUE THREAD

**Issue by orangeandblack5** [leadweedy/Firefox-Proton-Square#20]
Would really like colored bars to be added to the bottom of tabs in different containers like in pre-Proton Firefox, so you can tell at a glance what container something is in.

Right now it's impossible to tell at a glance, as far as I can tell, as it seemingly requires you to have a tab active since it's just the bar at the top of the active tab that gets recolored - and if you chose the default blue container group color good luck telling #37ADFF apart from #0A84FF.

Basically, I think this was really good:

![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/35876055/a7a8a9e2-0235-4079-aea0-1a81104a04bf)

**Comment by leadweedy** [comment]
that's a great suggestion I hadn't thought about, since I don't use containers extensively. since its a mozilla addon, I do think its worth supporting.


There's a couple design options so far:

1. australis/quantum style tabs of container colored stripe on bottom + static stripe on top
![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/72583854/533dfab5-b318-4d0e-bc00-f0d15b32a0c1)
its not the prettiest with the colors clashing but this stays true to the old design
![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/72583854/b3a006a4-bf25-47dd-a318-65604cb88d79)
per https://blog.mozilla.org/security/2020/02/06/multi-account-containers-sync/

2. make top and bottom stripes dynamic based on container color
![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/72583854/f3924dc1-5800-498a-8957-91450b1a4e24)
this is the option im leaning towards, as it keeps the same style as 1, but with more consistent color choice

4. full border around selected tab
![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/72583854/65a69b39-257e-4238-a5da-1da0a46b0d7d)
im not the biggest fan of this, since its a bit too prominent on the UI

**Comment by leadweedy** [comment]
variation on 2 with a 1px height
![image](https://github.com/leadweedy/Firefox-Proton-Square/assets/72583854/922f3213-4143-4e6a-8aa2-c2b198e511d8)

**Comment by orangeandblack5** [comment]
Personally I think either of 1 or 2 would probably be ideal, but don't strongly care which ended up being final because both address the issue and beyond that point it's mostly preference.

Technically, 1 is more consistent for communicating info to the user - the blue stripe **always** appears above your active tab and is never any other color, so personally I'd probably do that, but I'm more utilitarian so the potential for color clash doesn't bother me that much (and if it did I could always just adjust my container colors to alleviate the issue).

I would not do 3.

Thanks for the quick experimenting on this by the way, really cool to see, especially since multi-account containers is such a standout feature of Firefox.

**Comment by leadweedy** [comment]
implemented with https://github.com/leadweedy/Firefox-Proton-Square/pull/21

**Comment by orangeandblack5** [comment]
ended up changing it to 2px on my end since I use containers so frequently and it's a bit more readable at a glance with a bit more thickness, but on the whole this is fantastic thanks much
